src/Microsoft.AspNetCore.Mvc.ApiExplorer/ApiResponseTypeProvider.cs

```diff
@@ -172,6 +176,15 @@ private static List<IApiResponseMetadataProvider> GetResponseMetadataAttributes(
                             });
                         }
                     }
+
+                    if (!isSupportedContentType && contentType != null)
+                    {
+                        // No output formatter was found that supports this content type. Add the user specified content type as-is to the result.
+                        apiResponse.ApiResponseFormats.Add(new ApiResponseFormat
+                        {
+                            MediaType = contentType,
```

Comment:
`ProducesAttribute` does not support wildcards - https://github.com/aspnet/Mvc/blob/release/2.2/src/Microsoft.AspNetCore.Mvc.Core/ProducesAttribute.cs#L127-L131. 
I'm not doing anything special with wildcards - seems like a narrow scenario to account for.